### PR TITLE
Add photo signup

### PR DESCRIPTION
**Pull request by kanghsieh**
added photo, first/last name, nickname, age, gender, gym selection, quote to sign-up process.

new user is now being created correctly after signup-process and can see his own profile afterwards.

customized user device registrations controller with help of following reference: [Link](https://dev.to/casseylottman/adding-a-field-to-your-sign-up-form-with-devise-10i1)

### CODE REVIEW

File: app/controllers/users/registrations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
```
Comment (kanghsieh) [inline app/controllers/users/registrations_controller.rb:3]: created users registrations controller to customize the user sign up process of devise.

File: app/models/user.rb [hunk 2]
```diff
@@ -22,7 +22,7 @@ class User < ApplicationRecord
   has_many :users_weekdays, dependent: :destroy
   has_many :weekdays, through: :users_weekdays
 
-  has_one_attached :photo
+  has_one_attached :photo, dependent: :destroy
```
Comment (kanghsieh) [inline app/models/user.rb:25]: makes sure that users can be destroyed while destroying the photo at the same time

File: config/routes.rb [hunk 3]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  devise_for :users
+  # devise_for :users
+  devise_for :users, controllers: { registrations: 'users/registrations' }
```
Comment (kanghsieh) [inline config/routes.rb:3]: change the route to recognize the customized registrations controller

**Review by if-stella — COMMENTED** [review]
Looks good, ready to get merged and to be styled.

**Pull request merged**